### DISCUSSION TITLE
Network scan

### DISCUSSION THREAD

**Discussion by SandroCristiano** [wazuh/wazuh#13431] [Q&A] [answered]
Hello! Can Wazuh scan for vulnerabilities on the entire network without the need to install the agent in every single computer? Here's the situation: in one of our clients we've install wazuh for vulnerability scans and installed agents on the servers, so far so good. Now, the client wants to scan his entire network and he claims no agent is necessary because that's how it used to be with Alienvault. I went over the documentation but I was unable to find a proper answer. Thanks.

**Comment by HanesSciarrone** [comment, marked as answer]
HI @SandroCristiano 

I hope you are well. Thanks for using Wazuh and sharing your question with the community. If you want to detect vulnerabilities in your system, the requirements are:

- Agent installed on the system to scan.
- Agent connected to the manager.

This is because the vulnerability detection process is:

1. The vulnerability detector asks for the agents connected to the manager and obtains the IDs.
2. It searches the Syscollector database for each agent ID.
3. After getting the packages, hotfixes (Only in windows) and OS data it run the vulnerability scan with this information.

I hope you find the information useful.

Best regards
Hanes
